packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -83,7 +83,11 @@ export const WelcomeLayout = ({ children }: WelcomeLayoutProps) => {
                     <Right
                         bannerSlot={
                             bannerMessage && (
-                                <MessageSystemBanner message={bannerMessage} margin={spacings.xs} />
+                                <MessageSystemBanner
+                                    message={bannerMessage}
+                                    margin={spacings.xs}
+                                    width="100%"
```

Comment:
This could be the default value.

Reply:
That was my first though, but one never knows regarding styling around here :). cc @jvaclavik

Reply:
100% as a default value could be safe here IMHO

Reply:
Well, `< MessageSystemBanner>` is used also in Suitebanners, OnboardingLayout.

Reply:
OK, let's keep it this way. It doesn't matter much and it can break something if we will change the behaviour.